src/cheri_insts.sail

```diff
@@ -1430,6 +1430,110 @@ function clause execute (LoadCapCap(cd, cs1)) = {
   handle_load_cap_via_cap(cd, 0b0 @ cs1, cs1_val, vaddr)
 }
 
+union clause ast = CLoadTags : (regidx, regidx)
+/*
+ * XXX: [LC] should be [LC](LoadCapImm) but that syntax does not generate
+ * valspec links, it always uses the argument literally.
+ */
+/*!
+ * Integer register *rd* is replaced with the tags of the capabilities located
+ * in memory at and above *cs1*.**address**. The 0th bit corresponds to the
+ * first capability in memory. The result is coherent with other processors, as
+ * if the corresponding data words had also been loaded. The number of tags
+ * loaded is implementation-defined; typical implementations are expected to
+ * return the tags held in an L1 cache line, and so we use the constant
+ * [caps_per_cache_line]. The number of tags loaded must be a power of two, at
+ * least 1, and no more than **XLEN**.
+ *
+ * ## Exceptions
+ *
+ * An exception is raised if:
+ *   - *cs1*.**tag** is not set.
+ *   - *cs1* is sealed.
+ *   - *cs1*.**perms** does not grant both **Permit_Load** and
+ *     **Permit_Load_Capability**.
+ *   - *cs1*.**address** $\lt$ *cs1*.**base**.
+ *   - *cs1*.**address** $+$ [caps_per_cache_line] $\times$ **CLEN** $/$ 8
+ *     $\gt$ *cs1*.**top**.
+ *   - *cs1*.**address** is unaligned.
+ *   - The page table entry for *cs1*.**address** would cause the tag to be
+ *     cleared.
+ *
+ * ## Notes
+ *
+ * - In order to reduce DRAM traffic, implementations may choose to load only
+ *   the tags and not the corresponding data, and may wish to not evict other
+ *   lines by treating it as either an uncached or a non-temporal/streaming
```

Comment:
"uncached" apparently means "skipping over any cached values" in some contexts and so might be confusing.  Maybe say "non-evicting" or something?

Reply:
I think I'll just drop the uncached part, non-temporal/streaming can encompass that behaviour.